dj_cqrs/management/commands/cqrs_consume.py

```diff
@@ -53,35 +114,78 @@ def add_arguments(self, parser):
             help='Choose model(s) by CQRS_ID for consuming',
         )
         parser.add_argument(
-            '--reload', '-r', help='Enable reload signal SIGHUP', action='store_true',
+            '--reload',
+            '-r',
+            help=(
+                'Enable reload signal SIGHUP and autoreload '
+                'on file changes'
+            ),
+            action='store_true',
+            default=False,
+        )
+        parser.add_argument(
+            '--ignore-paths',
+            nargs='?',
+            type=str,
+            help=(
+                'Specify directories to ignore, '
+                'to ignore multiple paths use a comma as separator, '
+                'e.g. "env" or "env,node_modules"'
+            ),
+        )
+        parser.add_argument(
+            '--sigint-timeout',
+            nargs='?',
+            type=int,
+            default=5,
+            help='How long to wait for the sigint timeout before sending sigkill.',
+        )
+        parser.add_argument(
+            '--sigkill-timeout',
+            nargs='?',
+            type=int,
+            default=1,
+            help='How long to wait for the sigkill timeout before issuing a timeout exception.',
         )
 
-    def handle(self, *args, **options):
-        if not options['workers'] and not options['reload']:
-            current_transport.consume(**self.get_consume_kwargs(options))
-            return
-
-        self.start_workers_pool(options)
+    def handle(
+            self,
+            *args,
+            workers=1,
+            cqrs_id=None,
+            reload=False,
+            ignore_paths=None,
+            sigint_timeout=5,
+            sigkill_timeout=1,
+            **options,
+    ):
+
+        paths_to_ignore = None
+        if ignore_paths:
+            paths_to_ignore = [Path(p).resolve() for p in ignore_paths.split(',')]
 
-    def start_workers_pool(self, options):
         workers_manager = WorkersManager(
-            options, current_transport, self.get_consume_kwargs(options),
+            workers=workers,
+            transport=current_transport,
+            consume_kwargs=self.get_consume_kwargs(cqrs_id),
+            reload=reload,
+            ignore_paths=paths_to_ignore,
+            sigint_timeout=sigint_timeout,
+            sigkill_timeout=sigkill_timeout,
         )
-        if options['reload']:
-            try:
-                multiprocessing.set_start_method('spawn')
-            except RuntimeError:
-                pass
 
-            signal.signal(signal.SIGHUP, workers_manager.reload)
+        signal.signal(signal.SIGHUP, workers_manager.reload)
```

Comment:
No need to register signal handlers here they have already been registered in the run function of the workers manager.